### PR TITLE
Add serial PK

### PR DESCRIPTION
**Pull request by shen-h**
- Add serial PK to column_info, numeric_column, text_column, date_column
    - Add FKs accordingly
- Add code_id as serial PK to code_frequency
    - Add boolean column is_null
    - Add column_id as a FK to column_info

### CODE REVIEW

**Comment by shen-h** [comment]
@mollyrossow While this is under your review, I'll continue to work on CLI based on the current dev branch.  Once we've finished editing the table definitions, I think the hanging tasks are modifying functions related to column_info in extract_metadata.py, handling null values in code_frequency, and writing tests for it. I'll let you know how CLI goes. Thanks.

**Comment by mollyrossow** [comment]
@shen-h I think the most recent design discussed over email does not include the column is_null. Instead, we are removing the not null constraint on the code column which should be taken care of since it's not longer a primary key. That way we can store an actual null. I'll forward you the email chain I'm thinking of.

**Comment by shen-h** [comment]
@mollyrossow Thanks for the reminder. That email makes sense to me. I removed `is_null` in the last commit. Thanks.

File: alembic/versions/0fbe9f4e9934_create_tables.py [hunk 1]
```diff
@@ -202,7 +203,7 @@ def upgrade():
 
     op.create_table(
         'numeric_column',
-        sa.Column('data_table_id', sa.Integer),
```
Comment (mollyrossow) [inline alembic/versions/0fbe9f4e9934_create_tables.py]: I think we should leave data_table_id in, just not as a primary key.

**Comment by mollyrossow** [comment]
These changes to the database break the rest of the code. I suggest making the additional required changes before we merge this into dev, especially since Drew and Jonathan might be taking a look at dev.

**Comment by shen-h** [comment]
Sure. I'm making the changes and will update shortly.

**Comment by shen-h** [comment]
@mollyrossow Empty values can now be found with `where code is null`. I think the update covered most of the problems, but I only tested against adrf-000087.csv since my VM didn't work with the rest of the sample data. It will be great if you can pick up the tests. Thanks.

Note that in this commit, `test_rollback` is temporarily removed. I'll continue thinking about that test.

**Comment by shen-h** [comment]
@mollyrossow I just resolved our merge conflict and checked flake8, the tests, alembic upgrade/downgrade, and the CLI `extract.py`. I think we are good to go. Thanks a lot!

**Pull request merged**